gnome-base/gnome-vfs/gnome-vfs-2.24.4-r5.ebuild

```diff
@@ -141,14 +132,14 @@ multilib_src_test() {
 	unset DISPLAY
 	# Fix bug #285706
 	unset XAUTHORITY
-	Xemake check
+	virtx emake check
 }
 
 multilib_src_install() {
 	gnome2_src_install
 }
 
 multilib_src_install_all() {
-	DOCS="AUTHORS ChangeLog HACKING NEWS README TODO"
+	local DOCS=( AUTHORS ChangeLog HACKING NEWS README TODO )
 	einstalldocs
 }
```

Comment:
just define these DOCS in global scope and nix `multilib_src_install_all`?

Reply:
That will make us define it somewhat ugly like that or otherwise it fails:

``DOCS=( "${S}"/{AUTHORS,ChangeLog,HACKING,NEWS,README,TODO} )``